core/src/main/scala/com/yahoo/maha/jdbc/package.scala

```diff
@@ -6,94 +6,19 @@
 
 package com.yahoo.maha
 
-import java.sql.{Date, ResultSet, Timestamp}
-import java.text.SimpleDateFormat
+import java.sql.ResultSet
 
 import scala.util.Try
 
 package object jdbc {
-
-  type Seq[+A] = scala.collection.immutable.Seq[A]
```

Comment:
isnt it used  anywhere?

Reply:
It wasn't used in tests, or even the maha-api-example file that was attempting to use it.  I built it, then I rebuilt CDW with this change, and nothing broke.  A lot of this was extra stuff from the GitHub credited in the jdbcConnection file.

Reply:
Did manage to find an issue with it.  The question is if we want to keep that specific definition inside that file or move it elsewhere.  Does it belong there?